specification/synapse/data-plane/readme.md

```diff
@@ -230,6 +230,7 @@ input-file:
   - Microsoft.Synapse/stable/2020-12-01/entityTypes/SqlScript.json
   - Microsoft.Synapse/stable/2020-12-01/entityTypes/Trigger.json
   - Microsoft.Synapse/stable/2020-12-01/entityTypes/Credential.json
+  - Microsoft.Synapse/stable/2021-04-01/symsAPI.json
```

Comment:
Please put it under tag `package-artifacts-composite-v7`, we will use that tag to generate SDK, thanks.

Reply:
Updated.